src/Custom/Audio/OpenAIAudioModelFactory.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenAI.Audio;
+
+/// <summary> Model factory for models. </summary>
+public static partial class OpenAIAudioModelFactory
+{
+    /// <summary> Initializes a new instance of <see cref="OpenAI.Audio.AudioTranscription"/>. </summary>
+    /// <param name="duration"> The duration of the input audio. </param>
+    /// <returns> A new <see cref="OpenAI.Audio.AudioTranscription"/> instance for mocking. </returns>
```

Comment:
I'm only adding docstrings to a parameter if its corresponding property also has a docstring, from where I copied the text.

Reply:
Don't worry about the docstrings for the parameters. <summary> and <returns> are more than enough for now.

Reply:
I'll be consistent and remove all docstrings for parameters then, so we don't forget any parameter when we add them all.

Reply:
Sounds good!